requirements.txt

```diff
@@ -5,3 +5,4 @@ transifex-client
 markdown
 path.py
 XBlock>=1.2.2
+bulk-grades
```

Comment:
extremely high-level, but I probably would rename this to make clear that it's only useful for open edX